src/videouplinkbandwidthpolicy/VideoUplinkBandwidthPolicy.ts

```diff
@@ -61,7 +61,7 @@ export default interface VideoUplinkBandwidthPolicy {
   updateConnectionMetric(metrics: ConnectionMetrics): void;
 
   /**
-   * Returns the selected [[MediaTrackConstraints]] to update
+   * This function is deprecated and unused, and will be removed in a future release.
```

Comment:
nit: Can we reformat as TODO? in major version, we usually just do a TODO search for where we need to remove.

Reply:
I think we banned TODOs at some point lol. I've been intentionally using the word 'deprecated' on these types of functions to make them easier to find in the future.